lib/NetPacks.h

```diff
@@ -729,13 +689,12 @@ struct GiveHero : public CPackForClient
 
 struct OpenWindow : public CPackForClient
 {
-	OpenWindow():id1(-1),id2(-1){}
 	void applyCl(CClient *cl);
 
 	enum EWindow {EXCHANGE_WINDOW, RECRUITMENT_FIRST, RECRUITMENT_ALL, SHIPYARD_WINDOW, THIEVES_GUILD,
 	              UNIVERSITY_WINDOW, HILL_FORT_WINDOW, MARKET_WINDOW, PUZZLE_MAP, TAVERN_WINDOW};
 	ui8 window;
-	si32 id1, id2;
+	si32 id1 = -1, id2 = -1;

```

Comment:
Kind of surprised - I saw readability check in clang-tidy for *local* variables, that removes multiple declarations in a single line. But nothing for same case in classes?

Reply:
formatting issues resolved AFAIK.